descriptors/ModuleDescriptor-template.json

```diff
@@ -415,7 +483,8 @@
         "invoice-storage.invoice-number.get",
         "invoice-storage.invoice-line-number.get",
         "voucher-storage.module.all",
-        "invoice-storage.acquisitions-unit-assignments.all"
+        "invoice-storage.acquisitions-unit-assignments.all",
+        "voucher-storage.acquisitions-unit-assignments.all"
```

Comment:
I think `voucher-storage.module.all` is a right place for sub permission `voucher-storage.acquisitions-unit-assignments.all`